priam/src/main/java/com/netflix/priam/health/InstanceState.java

```diff
@@ -225,8 +180,9 @@ public void setYmlWritten(boolean yml) {
 
     public static class RestoreStatus {
         private LocalDateTime startDateRange, endDateRange; // Date range to restore from
-        private LocalDateTime executionStartTime,
-                executionEndTime; // Start-end time of the actual restore execution
+        // Start-end time of the actual restore execution
+        // Note these are referenced when this class is serialized to a String.
+        private LocalDateTime executionStartTime, executionEndTime;
```

Comment:
At some point, we should remove LocalDateTime to Instant.

Reply:
Ack.